.circleci/config.yml

```diff
@@ -303,7 +303,8 @@ jobs:
           name: 'Aggregate test results'
           shell: bash.exe
           command: |
-            npm run aggregateJUnit
+            set -e
```

Comment:
Do we need `set -e` if the new junit script does exit with an error ?

Reply:
Yeah, CircleCI doesn't fail the step unless we set this even if the script exits with a non-zero exit code. https://support.circleci.com/hc/en-us/articles/115015733328-Step-should-fail-but-job-finished-successfully

Reply:
Got it, thanks for clarifying